### PR TITLE
Improve contact us feedback page texts

### PR DESCRIPTION
**Pull request by thomasmalbaux**
Some admin use the "Contact us" page hoping to reach Sharetribe support. 

This changes a bit some copy texts and add some info texts to make it more obvious that this is a form used by their users to contact them, and that they should not use it to contact Sharetribe support.

screenshot-aalto sharetribe us 2016-08-14 11-53-30

### CODE REVIEW

File: features/step_definitions/admin_manage_members_steps.rb [hunk 1]
```diff
@@ -106,9 +106,9 @@ def find_remove_link_for_person(full_name)
 
 Then(/^I should be able to send a message to admin$/) do
   steps %Q{
-    When I fill in "What would you like to tell us?" with "I sad that I have been banned."
-    And I press "Send feedback"
-    Then I should see "Thanks a lot for your feedback!" within ".flash-notifications"
+    When I fill in "Your message to %{service_name} team" with "I sad that I have been banned."
```
Comment (thomasmalbaux) [inline features/step_definitions/admin_manage_members_steps.rb]: Hum some help needed here to learn how to manage variables in tests.

Comment (bladealslayer) [inline features/step_definitions/admin_manage_members_steps.rb]: `I sad that I have been banned.` Why not make the test person a bit more happy :)

Comment (bladealslayer) [inline features/step_definitions/admin_manage_members_steps.rb]: @thomasmalbaux, change `"Your message to %{service_name} team"` to `"feedback_content"`. That should fix the tests and avoid having to work with variables here.

**Comment by bladealslayer** [comment]
Looks good. Squash commits and feel free to merge.

**Pull request merged**